src/checkout-buttons/create-checkout-button-initializer.ts

```diff
@@ -35,15 +35,15 @@ import createCheckoutButtonRegistry from './create-checkout-button-registry';
 export default function createCheckoutButtonInitializer(
     options?: CheckoutButtonInitializerOptions
 ): CheckoutButtonInitializer {
-    const host = options && options.host;
+    const { host, locale = '' } = { ...options };
```

Comment:
Should we default to something instead of empty string for `locale`?

Also, instead of during the spread on `options`, you could do `options ?? {};` which is the intention here